sim/mcuboot-sys/src/api.rs

```diff
@@ -1,104 +1,198 @@
 //! HAL api for MyNewt applications
 
 use simflash::{Result, Flash, FlashPtr};
-use lazy_static::lazy_static;
 use libc;
 use log::{Level, log_enabled, warn};
 use std::{
     collections::HashMap,
     mem,
-    ops::Deref,
     slice,
-    sync::Mutex,
 };
+use crate::area::CAreaDesc;
```

Comment:
Although the simflash one seems to be out of order, lets at least try to keep these in alphabetical order.

Reply:
I reordered all of them and added `std` stuff to the block.